### DISCUSSION TITLE
Where to integrate the github action?

### DISCUSSION THREAD

**Discussion by alejandrorto** [Legit-Labs/legitify#246] [Q&A] [answered]
Hello everyone, 

I am a security analyst and we are exploring integrating Legitify into our organization. At first our goal would be to run a complete analysis of our entire organization periodically. We have seen that it can be integrated in a Github Action.

We have a question about this, does the repository where it is located need any special permissions or configuration needs, or can it be in any repository? Are there other ways to integrate the action?

Thank you very much in advance.

**Comment by noamd-legit** [comment, marked as answer]
Hi @alejandrorto.
The repository you put the action in doesn't matter. Usually, users put it in a dedicated repository or within the '.github' repository.
You need to make sure the GITHUB_TOKEN you provide to the actions has sufficient permissions to analyze the entire organization.
